pdlua.c

```diff
@@ -1047,7 +1058,9 @@ static void pdlua_menu_open(t_pdlua *o)
 
 static t_int *pdlua_perform(t_int *w){
     t_pdlua *o = (t_pdlua *)(w[1]);
-    int nblock = (int)(w[2]);
+    t_signal *first_sig = (t_signal *)(w[2]);
+    int nblock = first_sig->s_n;
+    // FIXME: should we use a o->blocksize here that gets set in dsp?
```

Comment:
marking this comment i made in the code for actual review.

Reply:
replaced this with `x->blocksize` now in https://github.com/agraef/pd-lua/commit/59632795544475e76de84dd1f32f7e870e470d57